mlrun/feature_store/feature_set.py

```diff
@@ -533,6 +547,7 @@ def add_aggregation(
         :param state_name: *Deprecated* - use step_name instead
         :param after:      optional, after which graph step it runs
         :param before:     optional, comes before graph step
+        :param emit_policy: optional, which emit policy to use when performing the aggregations
```

Comment:
But what is an emit policy? This is user facing documentation.

Reply:
I didn't want to over-document it, since it's currently only usable in this specific flow, i.e. `EmitEveryEvent` for `Spark`. I think I'll add a doc detailing this specific use-case and that's it.

Reply:
Done.